cmp/grid/impl/ColumnHeader.js

```diff
@@ -71,6 +71,18 @@ export const columnHeader = hoistCmp.factory({
             headerName = xhColumn.headerName({column: xhColumn, gridModel});
         }
 
+        const showTooltipWhenElided = isNil(impl.xhColumn.headerTooltip),
```

Comment:
I believe that `impl.xhColumn` can be null - note that line 69 above checks for this - due to direct usage of agGrid features around auto grouping columns or something similar.  Would need to make this nullsafe as well.

Reply:
Thank you!  Fixed + cleaned up line 69.